src/app/containers/NewStoryPromo/PromoOrganisms/RelatedContentPromo/RelatedContentPromo.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Promo from '../../Promo';
+
+const RelatedContentPromo = () => {
+  return (
+    <Promo>
+      <Promo.Link>
```

Comment:
Should the link be here? Usually we have this nested inside the heading?

Reply:
Nope, it should not. I have not touched the Related Content Promo yet. That is just a template I did in 1 minutes 😅